### PR TITLE
docs: turborepo based readmes

### PR DESCRIPTION
**Pull request by MildTomato**
added and amended readme files for turborepo based setup

(wait for https://github.com/supabase/supabase/pull/9082 to merge first)

### CODE REVIEW

File: DEVELOPERS.md [hunk 1]
```diff
@@ -52,111 +54,70 @@ To contribute code to Supabase, you must fork the [Supabase Repository](https://
...
 
-   ```sh
-   npm run dev
-   ```
+For example:
 
-   or with yarn
+- `npm install @supabase/ui -w common`: installs into `./packages/common`
+- `npm install @supabase/ui -w www`: installs into `./apps/www`
+- `npm install @supabase/ui -w studio`: installs into `./studio`
```
Comment (MildTomato) [inline DEVELOPERS.md:108]: ` suggestion
- `npm install react -w common`: installs into `./packages/common`
- `npm install react -w www`: installs into `./apps/www`
- `npm install react -w studio`: installs into `./studio`
`

File: DEVELOPERS.md [hunk 2]
```diff
@@ -52,111 +54,70 @@ To contribute code to Supabase, you must fork the [Supabase Repository](https://
...
-   npm
+The format is: `npm install <package name> -w=<workspace to install in>`.
 
-   ```sh
-   npm run dev
-   ```
+For example:
 
-   or with yarn
+- `npm install @supabase/ui -w common`: installs into `./packages/common`
```
Comment (MildTomato) [inline DEVELOPERS.md:106]:
```suggestion
- `npm install react -w common`: installs into `./packages/common`
```

File: DEVELOPERS.md (hunk 1 above)
Comment (MildTomato) [inline DEVELOPERS.md:108]:
```suggestion
- `npm install react -w studio`: installs into `./studio`
```

File: DEVELOPERS.md [hunk 3]
```diff
@@ -52,111 +54,70 @@ To contribute code to Supabase, you must fork the [Supabase Repository](https://
...
+- `/packages/common`: Common React code, shared between all sites.
+- `/packages/config`: All shared config
+- `/packages/spec`: Generates documentation using spec files.
+- `/packages/tsconfig`: Shared Typescript settings
 
-### Supabase Studio
+#### Installing packages
 
-1. Start development server
+Installing a package with NPM workspaces requires you to add the `-w` flag to tell NPM which workspace you want to install into.
```
Comment (saltcod) [inline DEVELOPERS.md:100]: "You should install all dependencies for any app from the root directory, not from each app"  ?

File: DEVELOPERS.md [hunk 4]
```diff
@@ -2,44 +2,46 @@
...
-You need to install and configure the following dependencies on your machine to build Supabase:
+You need to install and configure the following dependencies on your machine to build [Supabase](https://supabase.com):
 
 - [Git](http://git-scm.com/)
 - [Node.js v16.x (LTS)](http://nodejs.org)
 - [npm](https://www.npmjs.com/) version 7+ or [Yarn](https://yarnpkg.com/)
 
+## Local development
+
+We are in the process of migrating this repository to monorepo, using [Turborepo](https://turborepo.org/docs).
```
Comment (kiwicopple) [inline DEVELOPERS.md:34]: Is this still relevant? Looks like we're all-in on Turbo now?

File: DEVELOPERS.md [hunk 5]
```diff
@@ -52,111 +54,70 @@ To contribute code to Supabase, you must fork the [Supabase Repository](https://
...
-- [Supabase Website](#supabase-website)
-- [Supabase Docs](#supabase-docs)
-- [Supabase Studio](#supabase-studio)
+Site | Directory | Scope name | Description | Local development server
+---- | --------- | ---------- | ----------- | ------------------------
+[supabase.com](https://supabase.com) | `/apps/www` | www | The main website | http://localhost:3000
+[app.supabase.com](https://app.supabase.com) | `/studio` | studio | Studio dashboard | http://localhost:8082
+[POC] Next.js based Docs site | `/apps/temp-docs` | temp-docs | Guides and Reference (Next.js based) | http://localhost:3001  
+[POC] Community forum | `/apps/temp-community-forum` | forum | GitHub Discussions | http://localhost:3002
+[POC] DEV articles site | `/apps/temp-community-tutorials` | tutorials | DEV articles (which community members can write) | http://localhost:3003
```
Comment (kiwicopple) [inline DEVELOPERS.md:79]: Should we hide these from Turbo for now? Makes it easier for contributors

**Review by kiwicopple — APPROVED** [review]

File: DEVELOPERS.md (hunk 4 above)
Comment (saltcod) [reply DEVELOPERS.md:34]: I think docs is the last holdout? 

CleanShot 2022-09-23 at 11 02 00@2x

File: apps/reference/DEVELOPERS.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+   ```sh
+   git clone https://github.com/<github_username>/supabase.git
+   ```
+
+1. Go to the Supabase directory:
+   ```sh
+   cd supabase
+   ```
+
+### Running docs
```
Comment (dannykng) [inline apps/reference/DEVELOPERS.md:45]:
```suggestion
### Run the docs locally
```

File: apps/reference/DEVELOPERS.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+We'll be running the [Supabase Docs](https://supabase.com/docs)
```
Comment (dannykng) [inline apps/reference/DEVELOPERS.md:47]:
```suggestion
```

File: apps/reference/DEVELOPERS.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+1. Build development server
```
Comment (dannykng) [inline apps/reference/DEVELOPERS.md:49]:
```suggestion
1. Build the development server
```

File: apps/reference/DEVELOPERS.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+   npm run build
+   ```
+
+   or with yarn
+
+   ```sh
+   yarn build
+   ```
+
+1. Start development server
```
Comment (dannykng) [inline apps/reference/DEVELOPERS.md:63]:
```suggestion
1. Start the development server
```

File: apps/reference/DEVELOPERS.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+
+   ```sh
+   yarn start
+   ```
+
+1. Access the local server in your web browser at http://localhost:3010/docs.
+
+---
+
+### Which files to edit?
```
Comment (dannykng) [inline apps/reference/DEVELOPERS.md:81]:
```suggestion
### Editing files
```

File: apps/reference/DEVELOPERS.md [hunk 11]
```diff
@@ -0,0 +1,106 @@
+# Developing Supabase Docs
+
+1. [Development setup](#development-setup)
+   - [Install dependencies](#install-dependencies)
+   - [Fork the repository](#fork-the-repository)
+2. [Build Supabase](#build-supabase)
+   - [Running docs](#running-docs)
```
Comment (dannykng) [inline apps/reference/DEVELOPERS.md:7]:
```suggestion
   - [Run the docs locally](#run-the-docs-locally)
   - [Editing files](#editing-files)
```

File: DEVELOPERS.md (hunk 4 above)
Comment (MildTomato) [reply DEVELOPERS.md:34]: When that changes, lets adjust our docs to be turborepo focussed ?

**Comment by MildTomato** [comment]
@dannykng committed all your suggestions - looked good to me

**Review by dannykng — APPROVED** [review]

File: DEVELOPERS.md (hunk 5 above)
Comment (MildTomato) [reply DEVELOPERS.md:79]: @saltcod @joshenlim @alaister just do another PR if this isn't right. 
going to just merge this for now.

Comment (MildTomato) [reply DEVELOPERS.md]: yep, we've hidden these now

**Pull request merged**